### DISCUSSION TITLE
Can we make cipi version but in a docker images?

### DISCUSSION THREAD

**Discussion by bijancot** [andreapollastri/cipi#358] [Ideas]
I just wondering is there any thoughts about running cipi inside docker container. So we can also start or installing cipi on a server that already running and not missing something.

Regards,
Baskoro

**Comment by andreapollastri** [comment]
Cipi is a simple Laravel project with Laravel Sail (that is Docker based), you can install it with php artisan sail:install and run it via Docker or pull your image! 

The problem is that cipi has to work with real VPS or server to configure sites and other stuff via ssh.

**Reply by bijancot** [reply]
I see, so the problem here is how the container can communicate with the VPS. I think it's worth to try xd

Thank you for the response @andreapollastri !

**Comment by andreapollastri** [comment]
Hello guys, it the last 3 years the world has been changed with AI and with new deployments needs so a new "Laravel Control Panel" is not the real solution for today's world. I have created a new stable version of Cipi full of features and based on a powerful and light CLI. If you would like to try it, here you can find all informations: https://cipi.sh/
